dockstore-webservice/src/main/resources/migrations.1.15.0.xml

```diff
@@ -283,4 +283,19 @@
             </column>
         </addColumn>
     </changeSet>
+    <changeSet author="hyunnnaye" id="removeAuthorAndEmailInEntry">
+        <sql dbms="postgresql">
+            /* delete author and email column in an entry */
+            ALTER TABLE workflow DROP COLUMN author;
```

Comment:
This looks like you wrote it yourself, just FYI for the future, you can use the [propose_migrations.sh script](https://github.com/dockstore/dockstore/blob/8dc701e48b6ee401b7ef8fc069e20fb723a18cff/propose_migration.sh) to automatically generate migrations based on the JPA changes you made, i.e. it'll see you removed the author and email fields in several classes and generate a migration for it. Here's a wiki page on it https://wiki.oicr.on.ca/display/DOC/Database+Migrations